doc/src/build/programming-with-objects/ch2-using-objects.md

```diff
@@ -124,7 +124,7 @@ The first part is the same as what we have seen in [Chapter 1](./ch1-object-basi
 #### Option 2. Transfer the object
 The owner of the object may want to transfer it to another address. To support this, the `ColorObject` module will need to define a `transfer` API:
 ```rust
-public entry fun transfer(object: ColorObject, recipient: address) {
+public entry fun transfer(object: ColorObject, recipient: address, _ctx: &mut TxContext) {
```

Comment:
I think what we actually want here is to change `ColorObject` example (which should no longer need to manually define `transfer` because it has `store`.